files-community/Files
# Code Quality: Created PreviewPopupService

**Resolved / Related Issues**
   Related to #8052 and #12432

**Validation**
How did you test these changes?
   1. Install Quick Look
   2. Use spacebar to launch the preview

**Screenshots (optional)**
Add screenshots here.
